src/widgets/buttons.ts

```diff
@@ -478,6 +488,42 @@ export function deleteButtonWithCheck (
   return deleteButtonElt
 }
 
+function getButtonStyle (options: ButtonWidgetOptions = { buttonType: ButtonType.Primary, filled: true }) {
+  // We need to accomadate for legacy code, which is why we have to allow buttonType and filled to be optional
+  let backgroundColor: (ButtonType | string | undefined) = options.buttonType
+  let fontColor: (ButtonType | string | undefined) = '#ffffff'
+  let borderColor: (ButtonType | string | undefined) = options.buttonType
+  let hoverBackgroundColor: (ButtonType | string | undefined) = `lighten(${options.buttonType}, 5%)`
```

Comment:
this lighten function here doesn't seem to work.  I could try to make this work or I could just set it to the color it would be if it were lightened by 5%...

Reply:
yeah just set it to the colour.